x-pack/legacy/plugins/siem/dev_tools/circular_deps/run_check_circular_deps_cli.js

```diff
@@ -18,11 +18,12 @@ run(
     });
 
     const circularFound = result.circular();
-    if (circularFound.length !== 0) {
+    // We can only care about SIEM code, we should not be penalyze for others
+    if (circularFound.filter(cf => cf.includes('siem')).length !== 0) {
```

Comment:
Why don't we want to constrain this to just `siem` any more?

Reply:
because since we are importing outside of SIEM, I was getting a circular dependency on the `src` path where I do not have any control of.